pkg/network/telemetry/utils.go

```diff
@@ -0,0 +1,78 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package telemetry
+
+import (
+	"sort"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/util/common"
+)
+
+func contains(want string, strings []string) bool {
+	for _, candidate := range strings {
+		if want == candidate {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
let use community libraries for common methods like https://github.com/thoas/go-funk and `funk.ContainsString`

Reply:
I don't object to the idea, but I don't see this library being listed as a dependency (or even a transitive one), so I think it's a bit overkill to introduce one in this case. If you know of one that is already available, please let me know